rkt/trust.go

```diff
@@ -66,7 +66,12 @@ func init() {
 func runTrust(args []string) (exit int) {
 	if flagPrefix == "" && !flagRoot {
 		if len(args) != 0 {
-			stderr("--root required for non-prefixed (root) keys")
+			rootKeyName := args[0]
+			stderr(`Prevented potentially dangerous unbounded trust of all containers signed by keys from "%s"
+To contintue trusting this root domain:
+  rkt %s --root "%s"
+Or, establish a smaller scope with a prefix:
+  rkt %s --prefix "example.com/hello" "%s"`, rootKeyName, cmdTrustName, rootKeyName, cmdTrustName, rootKeyName)
```

Comment:
Can we include the two examples of smaller scopes from #480?

```
Or, establish a smaller scope with a prefix:
  rkt trust --prefix coreos.com/foo coreos.com
  rkt trust --prefix coreos.com/foobar/* coreos.com
```

This one extra line teaches you:
 - how a prefix is named
 - a prefix supports multiple levels of specificity
 - a prefix can target a specific ACI
 - a prefix supports wildcards